frontend/src/components/_shared/DashboardDatasetCard.tsx

```diff
@@ -16,119 +24,126 @@ export interface DashboardDatasetCardProps {
-            variant="success"
-          />
-        </div>
-      ) : (
-        <div className="h-8 w-8">
-          <Badge
-            className="flex items-center p-1"
-            icon={
-              <svg
-                width="24"
-                height="24"
-                viewBox="0 0 48 48"
-                fill="none"
-                xmlns="http://www.w3.org/2000/svg"
-              >
-                <path
-                  d="M7.2002 28.8V36C7.2002 39.9768 14.7218 43.2 24.0002 43.2C33.2786 43.2 40.8002 39.9768 40.8002 36V28.8C40.8002 32.7768 33.2786 36 24.0002 36C14.7218 36 7.2002 32.7768 7.2002 28.8Z"
-                  fill="#7E3AF2"
-                />
-                <path
-                  d="M7.2002 16.8V24C7.2002 27.9768 14.7218 31.2 24.0002 31.2C33.2786 31.2 40.8002 27.9768 40.8002 24V16.8C40.8002 20.7768 33.2786 24 24.0002 24C14.7218 24 7.2002 20.7768 7.2002 16.8Z"
-                  fill="#7E3AF2"
-                />
-                <path
-                  d="M40.8002 12C40.8002 15.9768 33.2786 19.2 24.0002 19.2C14.7218 19.2 7.2002 15.9768 7.2002 12C7.2002 8.02319 14.7218 4.79999 24.0002 4.79999C33.2786 4.79999 40.8002 8.02319 40.8002 12Z"
-                  fill="#7E3AF2"
-                />
-              </svg>
-            }
-            variant="purple"
-          />
-        </div>
-      )}
-      <div className="space-y-2 text-sm">
+      <div className="flex h-8 w-8 flex-col items-center gap-32 lg:flex-row lg:gap-8">
+        <Badge
+          className="flex h-8 w-8 items-center justify-center pl-1.5 pr-1.5"
+          icon={badgeIcon}
+          variant={badgeVariant}
+        />
+      </div>
+
+      <div className="w-full space-y-2 text-sm">
         <div className="flex items-center justify-between gap-4">
           <h2 className="text-lg font-bold">{title}</h2>
-          {tdcValidated && (
+          {tdc_category === "tdc_formatted" && (
             <Badge variant={"success"} className="text-[#03543F]">
-              TDC Validated
+              TDC Formatted
             </Badge>
           )}
+          {tdc_category === "tdc_harmonized" && (
+            <Badge variant={"warning"}>TDC Harmonized</Badge>
+          )}
         </div>
+
         <div className="flex gap-2">
-          {(keywords || [])?.map((k) => (
+          {(tags || [])?.map((k) => (
             <Badge variant={"purple"} className="bg-[#E5EDFF] text-[#42389D]">
-              {k}
+              {capitalize(k.display_name ?? "")}
             </Badge>
           ))}
         </div>
 
         <div className="flex flex-col gap-2 text-xs md:flex-row md:items-center">
-          <Badge
-            variant={"success"}
-            className="capitalize"
-            icon={
-              visibility === "public" ? (
+          {state === "draft" ? (
+            <Badge
+              variant={"success"}
+              className="capitalize"
+              icon={
                 <svg
```

Comment:
For code readability purposes, I would suggest moving this SVG to a separate component. Perhaps it can be a separate component in this same file.

Reply:
used hereicon EyeSlash icon instead.